services/fluent-bit/0.20.8/fluent-bit.yaml

```diff
@@ -14,7 +13,7 @@ spec:
         namespace: kommander-flux
       # This chart version should be kept in sync with logging-operator.
       # Both services should deploy the same version of fluent-bit.
-      version: 0.20.5
+      version: 0.20.8
```

Comment:
We should combine this PR with https://github.com/mesosphere/kommander-applications/pull/651.